task-features/daniel/labels.feature

```diff
@@ -0,0 +1,35 @@
+#cual es el objetivo de las pruebas
+Feature: Trello labels API
+    As a user I want to work with labels in the API
+
+  Scenario: Get data from specific label
+     Given the user does a GET request to /labels/id endpoint
+      When the user sends the request

```

Comment:
with `does` you are indicating that the request is send, so on this case the When doesn't have sense
```suggestion
     Given the user does a GET request to /labels/id endpoint
```
If want to keep the When, then use `defines` instead of `does`